.circleci/src/jobs/native_deliver_ios.yml

```
@@ -26,17 +26,19 @@ steps:
   - restore_cache:
       name: Restore Yarn Package Cache
       keys:
-        - 3-yarn-{{ arch }}-{{ checksum "yarn.lock" }}
+        - 3-yarn-{{ arch }}-{{ checksum "yarn.lock" }}-{{ checksum "package.json" }}
         - 3-yarn-{{ arch }}-
   - run:
       name: Install Dependencies
       command: yarn install --frozen-lockfile
   - save_cache:
       name: Save Yarn Package Cache
-      key: 3-yarn-{{ arch }}-{{ checksum "yarn.lock" }}
+      key: 3-yarn-{{ arch }}-{{ checksum "yarn.lock" }}-{{ checksum "package.json" }}
```

We can reuse a cache here:

5-yarn-{{ arch }}-{{ checksum "yarn.lock" }}-{{ checksum "package.json" }}
```suggestion
      key: 5-yarn-{{ arch }}-{{ checksum "yarn.lock" }}-{{ checksum "package.json" }}
```